R/get_employee.r

```diff
@@ -0,0 +1,96 @@
+#' Get employee data by specifying a set of fields.
+#'
+#' This is suitable for getting basic employee information, including current
+#' values for fields that are part of a historical table, like job title,
+#' or compensation information. See the fields endpoint for a list of possible
+#'  fields.
+#'
+#'
+#' @param id Employee IDs (character or numeric). The default of "directory"
+#' will give a directory of all employees, along with a table of possible fields.
+#' The special employee ID of zero (0) means to use the employee ID associated
+#' with the API key (if any).
+#'
+#' @param fields Character vector of fields to return. To see all available fields,
+#' see the column names of \code{get_employee()}.
+#'
+#' @param only_current Setting to false will return future dated values from
+#' history table fields.
+#'
+#' @param verbose Logical, default FALSE. Whether to return the fields dataframe
+#' when [id = "directory].
+#'
+#' @return If \code{id} is given then a list of the fields and their values. If
+#' [force_df = TRUE] then this is converted to a dataframe. If
```

Comment:
mentions the force_df parameter here, but not implemented in code, think we decided it wasn't required, is it just the return section that needs updating?

Reply:
Yeah good spot. Fixed and pushed.